src/index.js

```diff
@@ -16,9 +16,12 @@ module.exports = function plugin (options) {
       debug('process file: %s', file)
 
       // add file path info
+      var extname = path.extname(file)
+      var basename = path.basename(file)
```

Comment:
you should be able to pass `extname` as the [second parameter](https://nodejs.org/api/path.html#path_path_basename_p_ext) here, which elevates the need for `.replace` below

example:
```
path.basename('/foo/bar/baz/asdf/quux.html', '.html')
```